doc/doxygen/Doxyfile

```diff
@@ -851,7 +851,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = ../../include/pagmo/external ../../include/pagmo/detail
+EXCLUDE                = ../../include/pagmo/external ../../include/pagmo/detail/cec2013_data.hpp
```

Comment:
nah, thisis only a temp sol to explore doxygen warnings when the whole hv headers were there too.